include/seqan3/argument_parser/argument_parser.hpp

```diff
@@ -551,18 +553,18 @@ class argument_parser
             {
                 format = detail::format_help{false};
                 init_standard_options();
-                return;
+                special_format_was_set = true;
```

Comment:
It seems that if you pass multiple special formats (e.g., h and hh), it will overwrite it in the next loop iteration. I think we should either print an error message or always take the first one. Currently it would choose the last argument.